test/test_helper.rb

```diff
@@ -20,6 +20,7 @@
 
 # Do not allow network connections and external processes
 WebMock.disable_net_connect!(allow_localhost: true)
+WebMock.allow_net_connect!(net_http_connect_on_start: true)
```

Comment:
This sounds to me, we would completely allow network connection. That's probably not what we want.